src/app/FeedbackForm.js

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import styled from 'styled-components';
+
+const StyledQuestion = styled.div`
+  display: grid;
+  grid-template-rows: 1fr 1fr;
+  color: rebeccapurple;
+`;
+
+const StyledResponse = styled.div`
+  display: flex;
+  margin-bottom: 20px;
+  color: rebeccapurple;
+  font-size: 0.6rem;
+`;
+
+const StyledButton = styled.button`
+  margin-bottom: 5px;
+  padding: 5px;
+  background: rgba(218, 204, 62, 1);
+  color: white;
+  border: 1px solid rgba(218, 204, 62, 1);
+  font-size: 1.3em;
+`;
+
+const StyledForm = styled.form`
+  padding: 10px;
+`;
+
+export default function FeedbackForm({ questions, onSubmit, history }) {
+  return (
+    <StyledForm onSubmit={event => onSubmit(event, history)}>
+      <StyledQuestion>
+        Q{questions[0].question_id + 1}. {questions[0].question_text_en}
+      </StyledQuestion>
+      <StyledResponse>
+        <div>
```

Comment:
do you need the `div` or can you style the label as you wish?

Reply:
The form needs some more work, but I'll get to that when I add the other questions